plugin-management-cli/src/main/resources-filtered/.properties

```diff
@@ -1,2 +1,3 @@
 project.version = ${project.version}
```

Comment:
this file should probably move to the library now if we need it further down we shouldn't just duplicate it

Reply:
It was already in the `plugin-management-cli` folder, I created a new Version.properties in library, I don't know where .properties is being used.

Reply:
search for it

Reply:
I see, the file is used to print out the version of the tool.